### PR TITLE
Jetbrains Plugin: Minor changes

### PR DESCRIPTION
**Pull request by RohanVB**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: data/projects/jetbrains-ide-plugin.md [hunk 1]
```diff
@@ -33,14 +33,14 @@ So The project is divided into two parts:
...
 
 * A working plugin with minimal functionality exists.
-* Corresponding test cases exists.
-* Corresponding documentation exists.
 
 ##### CODING PHASE 2
 
-
+* Corresponding test cases exist.
+* Corresponding documentation exists.
```
Comment (sims1253) [inline data/projects/jetbrains-ide-plugin.md]: This should stay in phase 1. We could put the general stuff in here. this part from above "Have a general mechanism of highlighting code analysis results from external tools." needs a few milestones but thats what is missing

**Comment by RohanVB** [comment]
If anyone can take over this, that'd be great, I won't be able to make the deadline on this. 
Will look into it afterwards if its not done. :D

**Comment by jayvdb** [comment]
@rohanVB, can you undo the change as @sims1253 requested, rebase, and then we can ack and get this merged.
They are only minor changes, but they are good changes.

**Comment by RohanVB** [comment]
done @jayvdb

**Comment by hemangsk** [comment]
ack e8cebfd

**Comment by hemangsk** [comment]
@rultor merge

**Pull request merged**